tests/test_contiguous_layouts.cpp

```diff
@@ -335,7 +357,39 @@ struct TestLayoutConversion<
   }
 };
 
-TYPED_TEST_SUITE(TestLayoutConversion, layout_conversion_test_types);
+// We can't reuse TestLayoutConversion directly for multiple suites,
+// because this results in duplicate test names.
+// However, creating an alias for each suite works around this.
+
+TYPED_TEST_SUITE(TestLayoutConversion, layout_conversion_test_types_left_to_left);
```

Comment:
Why not use an alias for this too, so that the testnames show up systematically?

Reply:
Mark said nvc++ fails if he uses an alias here too.

Reply:
Right -- not sure why.  I'll experiment a bit once PR #147 lands.  Thanks!